papers.html

```diff
@@ -135,9 +135,33 @@
         </ul>
       </div>
     </nav>
-    
 
     <div class="container p-4">
+      <p></p>
+      <h5>Papers from our collaboration that showcase POSYDON and related work can be found at the links provided below, hosted on NASA ADS services and the Association for Computing Machinery (ACM) Digital Library.</h5>
+      <p></p>
+    </div>
+    <div class="container p-4">
+      <h1><a href="https://ui.adsabs.harvard.edu/public-libraries/ZZsD9bzLTzWnLV3hwyJxbA" target="_blank">NASA ADS Library</a></h1>
+    </div>
+    <div class="container p-4">
+      <h1>ACM Digital Library</h1>
+      <p>
+        <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#xtgt2021demo2" aria-expanded="false" aria-controls="xtgt2021demo2">
+          Direct links
```

Comment:
I think it is kind of awkward to have a dropdown here. How does it look if we just put the links directly below "ACM Digital Library"?

Reply:
Thanks for the feedback. I have updated this as suggested. How's it look?